### PR TITLE
Fixing the README.md and adding issues filing guide

### PR DESCRIPTION
**Pull request by blackdwarf**
The README.md contained some errors, corrected those. Removed the
native compilation bits for now.
Added an issue filing guide to make the issue process slightly better so we can
triage better.

/cc @piotrpMSFT

### CODE REVIEW

File: Documentation/issue-filing-guide.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+Filing issues for .NET Core CLI
+===============================
+
+As you may notice based on our issues page, the CLI repo is what is known as a 
+"high-profile" and "high-volume" repo; we 
+get a lot of issues. This, in turn, may mean that some issues get 
+lost in the noise and/or are not reacted on with the needed speed. 
+
+In order to help with the above situation, we need to have a certain way to file 
+issues in order for the core team of maintainers can react as fast as (humanly)
```
Comment (livarcocc) [inline Documentation/issue-filing-guide.md]: I think you need to fix this phrasing. ...in order for the core team of maintainers to react as fast as...

File: Documentation/issue-filing-guide.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+possible.
+
+The below steps are something that we believe is not a huge increase in process, 
+but would help us react much faster to any issues that are filed. 
+
+1. Check if the [known issues](known-issues.md) cover the issue you are running 
+into. We are collecting issues that are known and that have workarounds, so it 
+could be that you can get unblocked pretty easy.
```
Comment (livarcocc) [inline Documentation/issue-filing-guide.md]: should this be pretty easily?

File: Documentation/issue-filing-guide.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+the "bug" label, if it is a suggestion for a feature, use the "enhancement" 
+label. This helps the team get to defects more effectively. 
+
+3. Unless you are sure in which milestone the issue falls into, leave it blank.
+
+4. If you don't know who is on point to fix it or should be on point, assign it 
+first to @blackdwarf and he will triage it from there. 
+
+5. /cc the person that the issue is assigned to (or @blackdwarf) so that person 
+would get notified. In this way
```
Comment (livarcocc) [inline Documentation/issue-filing-guide.md]: Seems like you got interrupted here.

File: README.md [hunk 4]
```diff
@@ -53,19 +60,8 @@ Then you can either run from source or compile the sample. Running from source i
...
-	dotnet build --native
-
-The following command will perform native compilation using the C++ Codegenerator:
 
-    dotnet build --native --cpp
-
-If you are in Windows, make sure that you run the above command inside the *VS 2015 x64 Native Tools* prompt, otherwise you will get errors. This command will drop a native single binary in `./bin/[configuration]/[framework]/native/[binary name]` that you can run.
-
-**Note:** At this point, only the `helloworld` and `dotnetbot` samples will work with native compilation.
+This will drop a binary in `./bin/[configuration]/[framework]/[binary name]` that you can just run.
```
Comment (livarcocc) [inline README.md]: There will be a rid here if this has the emitentrypoint: true set in the PJ.

File: Documentation/issue-filing-guide.md (hunk 1 above)
Comment (blackdwarf) [inline Documentation/issue-filing-guide.md]: @livarcocc is the "(humanly)" the problem?

File: Documentation/issue-filing-guide.md (hunk 3 above)
Comment (blackdwarf) [inline Documentation/issue-filing-guide.md]: Ouch, that is true. :)

File: README.md (hunk 4 above)
Comment (blackdwarf) [inline README.md]: @livarcocc so `./bin/[configuration]/[framework]/[rid]/[binary name]`?

**Comment by blackdwarf** [comment]
@livarcocc new version up.

**Comment by blackdwarf** [comment]
@dotnet-bot test  CentOS7.1 Release Build please

**Comment by livarcocc** [comment]
:shipit:

File: Documentation/issue-filing-guide.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+5. /cc the person that the issue is assigned to (or @blackdwarf) so that person 
+would get notified. In this way the correct person can immediately jump on the 
+issue and triage it.
+
+6. For bugs, please be as concrete as possible on what is working, what 
+is not working. Things like operating system, the version of the tools, the 
+version of the installer and when you installed all help us determine the 
+potential problem and allows us to easily reproduce the problem at hand.
+
```
Comment (piotrpMSFT) [inline Documentation/issue-filing-guide.md]: For bugs, let's ask for a repo we can clone to repro:
What OS?
What Repo to clone?
What repro command to run from the root of that repo?

I know this won't always work, but will be really helpful in many cases.

File: README.md [hunk 6]
```diff
@@ -53,19 +60,8 @@ Then you can either run from source or compile the sample. Running from source i
...
-	dotnet build --native
-
-The following command will perform native compilation using the C++ Codegenerator:
 
-    dotnet build --native --cpp
-
-If you are in Windows, make sure that you run the above command inside the *VS 2015 x64 Native Tools* prompt, otherwise you will get errors. This command will drop a native single binary in `./bin/[configuration]/[framework]/native/[binary name]` that you can run.
-
-**Note:** At this point, only the `helloworld` and `dotnetbot` samples will work with native compilation.
+This will drop a binary in `./bin/[configuration]/[framework]/[rid]/[binary name]` that you can just run.
```
Comment (piotrpMSFT) [inline README.md]: This won't be true for much longer.

**Comment by blackdwarf** [comment]
@piotrpMSFT ups, I did not see your comments before merging. :( Will fix with another PR.

File: Documentation/issue-filing-guide.md (hunk 5 above)
Comment (blackdwarf) [inline Documentation/issue-filing-guide.md]: @piotrpMSFT we will add issue templates as well, so this will be a good way to add stuff you mentioned.

**Pull request merged**